crates/rpc-server/src/registry.rs

```diff
@@ -441,7 +444,9 @@ impl RequestSubmitter {
                 let state = snap.state().expect("get mem state");
                 let kind = req.kind();
                 let hash = req.hash();
-                match self.req_to_entry(req, &state, queue.len()) {
+                let dynamic_config_manager = &self.dynamic_config_manager.read().await;
+                let fee_config = dynamic_config_manager.get_fee_config();
+                match req_to_entry(fee_config, self.generator.clone(), req, &state, queue.len()) {
```

Comment:
In this background loop,
we should consider the time comsumed by `generator.clone()`.

Reply:
That should be fine. This clone is an Arc clone. And we only read backends here which is fast. We can consider cache backends in the future maybe.